lib/oban/job.ex

```diff
@@ -200,7 +200,7 @@ defmodule Oban.Job do
       |> Oban.insert()
   """
   @doc since: "0.1.0"
-  @spec new(args(), [option]) :: Ecto.Changeset.t()
+  @spec new(args(), [option]) :: changeset
```

Comment:
Purely for the sake of consistency:

```suggestion
  @spec new(args(), [option()]) :: changeset()
```

Reply:
Sure! I kept it like is because in some other places it's still like that. No problem at all :)